test/functional/rpc_blockchain.py

```diff
@@ -267,6 +267,13 @@ def _test_gettxoutsetinfo(self):
         # hash_type none should not return a UTXO set hash.
         res5 = node.gettxoutsetinfo(hash_type='none')
         assert 'hash_serialized_2' not in res5
+        assert 'muhash' not in res5
```

Comment:
bde3feba should `assert 'muhash' not in` be added to any of res-res4

Reply:
Done.